### ISSUE TITLE
RadioGroup Component - Accessibility compliance - unable to navigate and select with keyboard

### ISSUE THREAD

**Issue by tpGroupcare** [Megabit/Blazorise#2775]
When I use a RadioGroup then I experience various accessibility (a11y) problems dependent on whether or not the Buttons property is true or not. 

If it is true then I am unable to tab press my way through the radio buttons and ultimately select one without the use of the mouse. 

if it is false then I can tab press my way through the radio buttons but ultimately I am still unable to select one without the use of a mouse click. 

You can reproduce this on your own demo site. 

It would be preferable that one is able to tab press his/her way through the options of radio buttons even though the Buttons property is set to true in the component. 

Further it would be preferable that one can select a focused radio button with a key enter press.

I actually also found that for the Switch component you are also unable to toggle it with a key enter press. (This can also be reproduced on you own demo site.) 

The reason I am posting this as a feature request is because I am uncertain if this awesome but rather young project even concerns itself with accessibility issues yet.

**Comment by tpGroupcare** [comment]
It came to my attention that RadioGroups you can control with arrows. 

Switches you toggle with space and not enter.
